Korean translation update

revi opened the pull request:
Mostly proofreading, please refer to the commit messages for details.

Thanks!

revi committed:
Proofreading

Mostly space fixing

revi committed:
Another update

* SHA-1 was updated
* Spacing fix 
* Josa(postpositions) fix

revi committed:
Yet another fix

* Space fix
* Postpositions fix - some postpositions were using one form of
postpositions when they can be both of them.

nethip commented:
@revi Could you add any one who could do a linguistic review of Korean strings?

revi commented:
@ds5apn per his review on #8262

MarcelGerber commented:
@ds5apn @taggon @mixed @heisice Could you do a quick linguistic review of these changes?

taggon commented on src/nls/ko/strings.js:
```diff
@@ -54,26 +54,26 @@ define({
     "FILE_ALREADY_EXISTS": "{0} <span class='dialog-filename'>{1}</span>은 이미 존재 합니다.",
-    "ERROR_CREATING_FILE_TITLE": "{0}를 만들 때 에러가 발생했습니다",
-    "ERROR_CREATING_FILE": "{0} <span class='dialog-filename'>{1}</span>를 만들 때 에러가 발생했습니다.{2}",
+    "ERROR_CREATING_FILE_TITLE": "{0}를 만던 중에 에러가 발생했습니다",
```
This text should be "{0}을 만들던 중에 에러가 발생했습니다".

revi committed:
Fix typo

Pointed at Adobe/brackets#11101

taggon commented on src/nls/ko/strings.js:
```diff
@@ -92,26 +92,26 @@ define({
+    "LIVE_DEV_SERVER_NOT_READY_MESSAGE": "실시간 미리보기에 사용되는 HTTP 서버를 시작할 때 에러가 발생했습니다. 잠시 후에 다시 시도하세요.",
     "LIVE_DEVELOPMENT_INFO_TITLE": "실시간 미리보기에 오신 것을 환영합니다",
-    "LIVE_DEVELOPMENT_INFO_MESSAGE": "실시간 미리보기에 의해 {APP_NAME}가 브라우저에 연결했습니다. 브라우저에서 HTML 파일이 수정되면 미리보기가 즉시 업데이트됩니다. <br />이 초기 버전의 {APP_NAME}는 <strong> Google Chrome </strong>에서만 실시간 미리보기 기능하고 <strong> CSS 또는 HTML 파일 </strong>의 편집시에만 라이브 업데이트가 이루어집니다. JavaScript 파일에 대한 변경 사항은 저장시 자동으로 다시로드됩니다. <br /> (이 메시지는 한 번만 표시되지 않습니다.)",
+    "LIVE_DEVELOPMENT_INFO_MESSAGE": "실시간 미리보기에 의해 {APP_NAME}(가) 브라우저에 연결했습니다. 브라우저에서 HTML 파일이 수정되면 미리보기가 즉시 업데이트됩니다. <br />이 초기 버전의 {APP_NAME}은(는) <strong> Google Chrome </strong>에서만 실시간 미리보기 기능하고 <strong> CSS 또는 HTML 파일 </strong>의 편집시에만 라이브 업데이트가 이루어집니다. JavaScript 파일에 대한 변경 사항은 저장시 자동으로 다시로드됩니다. <br /> (이 메시지는 한 번만 표시되지 않습니다.)",
```
Because {APP_NAME} is always "Brackets", you don't need to use both two postpositions. The last korean sentence is mistranslated. Its corresponding english sentence is "You'll only see this message once", so it should say "이 메시지는 한 번만 표시됩니다". Here is my suggestion:

"{APP_NAME}가 실시간 미리보기를 통해 브라우저에 연결됐습니다. HTML 파일이 수정되면 브라우저에서 미리보기가 즉시 업데이트됩니다.<br /><br />현재 사용 중인 {APP_NAME} 초기 버전에서는 <strong>Google Chrome </strong>에서만 실시간 미리보기 기능하고 <strong>CSS 또는 HTML 파일</strong>을 편집할 때만 실시간으로 업데이트됩니다. 자바스크립트 파일에 대한 변경 사항은 저장할 때 자동으로 다시 로드됩니다.<br />(이 메시지는 한 번만 표시됩니다)"

taggon commented on src/nls/ko/strings.js:
```diff
@@ -164,7 +164,7 @@ define({
     "BUTTON_REGEXP_HINT": "정규식",
     "REPLACE_WITHOUT_UNDO_WARNING_TITLE": "복원할 수 없는 바꾸기",
-    "REPLACE_WITHOUT_UNDO_WARNING"      : "{0}개 이상의 파일을 수정해야하므로, {APP_NAME}은 열지 않은 파일도 수정할 것입니다.<br />열지 않은 파일에서 바꾼 내용은 복원할 수 없습니다.",
+    "REPLACE_WITHOUT_UNDO_WARNING"      : "{0}개 이상의 파일을 수정해야하므로, {APP_NAME}은(는) 열지 않은 파일도 수정할 것입니다.<br />열지 않은 파일에서 바꾼 내용은 복원할 수 없습니다.",
```
Just use "는" because {APP_NAME} is always "Brackets".

taggon commented on src/nls/ko/strings.js:
```diff
@@ -92,26 +92,26 @@ define({
     // Application error strings
     "ERROR_IN_BROWSER_TITLE": "{APP_NAME} 아직 브라우저에서 실행되지 않습니다.",
-    "ERROR_IN_BROWSER": "{APP_NAME}는 HTML로 구축되어 있지만 데스크톱 응용 프로그램으로 실행하여 로컬 파일을 편집 할 수 있습니다. {APP_NAME}을 실행하기 위해 <b>github.com / adobe / brackets-shell</b> 저장소 응용 프로그램 셸을 사용하십시오.",
+    "ERROR_IN_BROWSER": "{APP_NAME}은(는) HTML로 구축되어 있지만 데스크톱 응용 프로그램으로 실행하여 로컬 파일을 편집 할 수 있습니다. {APP_NAME}을 실행하기 위해 <b>github.com / adobe / brackets-shell</b> 저장소 응용 프로그램 셸을 사용하십시오.",
```
Don't use both two postpositions. `{APP_NAME}` will be replaced with `"Brackets"`.

taggon commented on src/nls/ko/strings.js:
```diff
@@ -92,26 +92,26 @@ define({
     "LIVE_DEVELOPMENT_ERROR_TITLE": "실시간 미리보기 에러",
     "LIVE_DEVELOPMENT_RELAUNCH_TITLE": "브라우저에 연결합니다",
-    "LIVE_DEVELOPMENT_ERROR_MESSAGE": "실시간 미리보기에 연결하려면 원격 디버깅을 사용하여 Chrome을 다시 시작해야합니다. <br /> Chrome을 다시 시작하고 원격 디버깅을 사용 하시겠습니까?",
+    "LIVE_DEVELOPMENT_ERROR_MESSAGE": "실시간 미리보기에 연결하려면 원격 디버깅을 사용하여 Chrome을 다시 시작해야 합니다. <br /> Chrome을 다시 시작하고 원격 디버깅을 사용 하시겠습니까?",
```
According to spelling system of Korean, `"사용 하시겠습니까?"` should be `"사용하시겠습니까?"`

taggon commented on src/nls/ko/strings.js:
```diff
@@ -494,11 +494,11 @@ define({
     "EXTENSION_ALREADY_INSTALLED": "이 패키지를 설치하면 이전에 설치한 확장 기능이 무시됩니다. 이전 확장을 덮어 쓰시겠습니까?",
     "EXTENSION_SAME_VERSION": "이 패키지 버전은 현재 설치되어있는 버전과 동일합니다. 기존 설치를 겹쳐 쓰시겠습니까?",
-    "EXTENSION_OLDER_VERSION": "이 패키지 버전 ({0})는 현재 설치되어있는 버전 ({1})보다 이전 버전입니다. 기존 설치를 겹쳐 쓰시겠습니까?",
+    "EXTENSION_OLDER_VERSION": "이 패키지 버전 ({0})은(는) 현재 설치되어있는 버전 ({1})보다 이전 버전입니다. 기존 설치를 겹쳐 쓰시겠습니까?",
```
In this case, the string will be like `"이 패키지 버전(1.2.2)은 현재..."`. That is, the first postposition will match `"버전"`. You should use `"은"` only instead of `"은(는)"`.

taggon commented on src/nls/ko/strings.js:
```diff
@@ -471,7 +471,7 @@ define({
     "OVERWRITE": "덮어쓰기",
     "CANT_REMOVE_DEV": "\"dev\" 폴더의 확장 기능은 수동으로 제거해야 합니다.",
-    "CANT_UPDATE": "업데이트가 이 버전의 {APP_NAME}과 호환되지 않습니다.",
+    "CANT_UPDATE": "업데이트가 이 버전의 {APP_NAME}과(와) 호환되지 않습니다.",
```
Please fix the postposition. I explained the reason before.

taggon commented:
@MarcelGerber Done. :)

revi committed:
Fix other errors

Pointed at adobe/brackets#11101

 - Fix double space introduced in previous commit
 - Fix josa errors on {APP_NAME}
 - Fix minor {} closing error
 - Other issues mentioned in the pullrequest

revi commented:
Fixing...

MarcelGerber commented:
@taggon: @revi made the changes you suggested. Is this fine with you now?

taggon commented:
Yes, it looks good to me.

MarcelGerber commented:
Thank you both.
Merging.
